web-client/src/styles/custom.scss

```diff
@@ -1984,3 +1984,17 @@ button.change-scanner-button {
   background-size: 30% 30%;
   outline: 3px solid #005ea2;
 }
+
+.text-right {
```

Comment:
🔧 There is already a text-right class in `utils.scss`!

Reply:
the utils.scss is only used for the pdf html generators, so we also needed it here I think.